### DISCUSSION TITLE
Header and sticky header

### DISCUSSION THREAD

**Discussion by PN34** [templaza/astroid-framework#354] [General]
Hi,

1) In the astroid header, I want to align the astroid-logo to the left. How could I do that?
2) I would also like to use the same menu mode for the sticky menu (astroid-logo above the menu). How could I do? Do I have to create my own module and to adapt the sticky animation?

This is the link: https://www.dev.kinesio-montpellier.fr
Thank you,

Philippe

**Comment by Chacapamac** [comment]
I @PN34 
**Do you mean something like** —> http://www.graf-test.ca/
Check logo “Graf” Top-Left and in the Sticky Header.

**Reply by PN34** [reply]
Thank you very much for your answer. 
I managed to get the logo on the left side.

But I would like to use the same header (stacked header mode) for sticky position, which is not available in the sticky header options.
Any idea what is the best way to do that ?

Thanks,

**Reply by Chacapamac** [reply]
@PN34 , for now the only way I can see is to override the Sticky header css coming from 
—> **media/templates/site/your_astroid_template/css/templates.css**

Also you have the sticky.php file that maybe can help?
—> **libraries/astroid/framework/frontend/header/sticky.php** 

You probably need to use flex and css to achieve what you want.

**ENHANCEMENT**
To have a way to do that in the Template Manager, only the addition of parameters and changes in the code will permit to have more control over the look of the Sticky Header.

**I Mark this as a Enhancement for** @sonvnn
